lib/topological_inventory/orchestrator/object_manager.rb

```diff
@@ -115,6 +115,15 @@ def delete_replication_controller(name)
       rescue Kubeclient::ResourceNotFoundError
       end
 
+      def get_collector_image(name)
+        pod = connection.get_pods(
+          :namespace => my_namespace,
+          :label_selector => "name=topological-inventory-#{name}-operations"
```

Comment:
I hooked up my local to the OCP4 API and there is not a `get_deployments` method generated like I thought there would be, the kubeclient gem metaprograms the methods based on what it discovers. Maybe it's different when running in OCP itself - but I can't exec in to figure it out for now. 

If that method does happen to exist though it will be as easy as changing it out and changing the `.spec.containers.first.image` -> `.spec.template.spec.containers.first.image`.